### PR TITLE
#96 Use Protobuf

### PR DESCRIPTION
**Pull request by Peyton-McKee**
Added support from protobuf, assuming all values are going to be sent as string characters

view argos #96 For more information and linked PR

https://github.com/Northeastern-Electric-Racing/Argos/pull/97/files#diff-218ade406544b9960ad10ab2ad61290e08e499a34daa2eb39f0ca42850429361

### CODE REVIEW

**Comment by jr1221** [comment]
Two questions:
- should we have the protobuf generated functions in git?
- are we getting rid of enum typing the unit field?

**Comment by Peyton-McKee** [comment]
I didnt see a purpose for it, the whole idea for the database is its as flexible as possible so i dont see any point for limiting units to an enum anymore

File: src/generated/serverdata/v1/serverdata.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/**
```
Comment (jr1221) [inline src/generated/serverdata/v1/serverdata.ts]: should we ignore these files as they are autogenerated?  argos already has a build hook that calls your script and it is probably best practice so nobody uses out of date versions of the typescript definitions.

Comment (Peyton-McKee) [reply src/generated/serverdata/v1/serverdata.ts]: sure i can ignore them

**Review by jr1221 — DISMISSED** [review]
might be good to note where the script has to be run from to generate correctly and find the dependencies. (the root of the parent project). But not an issue or anything.

**Pull request merged**